src/api/initialize.ts

```diff
@@ -9,6 +9,10 @@ const redirectToLogin = () => {
     window.location.href = isDevelopment() ? LOCAL_LOGIN : API_LOGIN;
 };
 
+const redirectToIngenTilgang = () => {
+    window.location.href = `${APP_ROOT}kandidat/ingen-tilgang`;
```

Comment:
Er ingen tilgang feilen bare ment for kandidatdetaljer siden?
Hvis ikke burde vi kanskje legge den siden på `/finn-kandidat/ingen-tilgang`

Reply:
Ehm what. Typo fra min side :p